server/src/test/java/org/elasticsearch/index/shard/IndexShardOperationPermitsTests.java

```diff
@@ -702,38 +689,28 @@ public void onResponse(Releasable releasable) {
                 public void onFailure(Exception e) {
                     throw new RuntimeException(e);
                 }
-            }, ThreadPool.Names.GENERIC, false, "");
+            }, ThreadPool.Names.GENERIC, false);
         };
     }
 
     public void testPermitTraceCapturing() throws ExecutionException, InterruptedException {
```

Comment:
This test seems to have been added in https://github.com/elastic/elasticsearch/pull/28567/ . Maybe it's not required at all now?

Reply:
Fair point, yes, this pretty much duplicates `testActiveOperationsCount` now. Removed.